javav2/example_code/entityresolution/src/main/java/com/example/entity/scenario/CloudFormationHelper.java

```diff
@@ -0,0 +1,188 @@
+            SdkAsyncHttpClient httpClient = NettyNioAsyncHttpClient.builder()
+                .maxConcurrency(100)
+                .connectionTimeout(Duration.ofSeconds(60))
+                .readTimeout(Duration.ofSeconds(60))
+                .writeTimeout(Duration.ofSeconds(60))
+                .build();
+
+            ClientOverrideConfiguration overrideConfig = ClientOverrideConfiguration.builder()
+                .apiCallTimeout(Duration.ofMinutes(2))
+                .apiCallAttemptTimeout(Duration.ofSeconds(90))
+                .retryStrategy(RetryMode.STANDARD)
+                .build();
+
+            cloudFormationClient = CloudFormationAsyncClient.builder()
+                .httpClient(httpClient)
+                .overrideConfiguration(overrideConfig)
+                .build();
+        }
+        return cloudFormationClient;
+    }
+
+    public static void deployCloudFormationStack(String stackName) {
+        String templateBody;
+        boolean doesExist = describeStack(stackName);
+        if (!doesExist) {
+            try {
+                ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+                Path filePath = Paths.get(classLoader.getResource(CFN_TEMPLATE).toURI());
+                templateBody = Files.readString(filePath);
+            } catch (IOException | URISyntaxException e) {
+                throw new RuntimeException(e);
+            }
+
+            getCloudFormationClient().createStack(b -> b.stackName(stackName)
+                    .templateBody(templateBody)
+                    .capabilities(Capability.CAPABILITY_IAM))
+                .whenComplete((csr, t) -> {
+                    if (csr != null) {
+                        System.out.println("Stack creation requested, ARN is " + csr.stackId());
+                        try (CloudFormationAsyncWaiter waiter = getCloudFormationClient().waiter()) {
+                            waiter.waitUntilStackCreateComplete(request -> request.stackName(stackName))
+                                .whenComplete((dsr, th) -> {
+                                    if (th != null) {
+                                        System.out.println("Error waiting for stack creation: " + th.getMessage());
+                                    } else {
+                                        dsr.matched().response().orElseThrow(() -> new RuntimeException("Failed to deploy"));
+                                        System.out.println("Stack created successfully");
+                                    }
+                                }).join();
+                        }
+                    } else {
+                        System.out.format("Error creating stack: " + t.getMessage(), t);
+                        throw new RuntimeException(t.getCause().getMessage(), t);
+                    }
+                }).join();
+        } else {
+            logger.info("{} stack already exists", CFN_TEMPLATE);
```

Comment:
This currently prints ``template.yaml stack already exists", which is incorrect.

```suggestion
            logger.info("{} stack already exists", stackName);
```

Reply:
Chaznged to stack name. Nice pickup.